src/Raven.Server/Commercial/LicenseManager.cs

```diff
@@ -182,8 +183,8 @@ public async Task PutMyNodeInfoAsync()
             }
             catch (Exception e)
             {
-                if (Logger.IsOperationsEnabled && _serverStore.IsPassive() == false)
-                    Logger.Operations("Failed to put my node info, will try again later", e);
+                if (Logger.IsDebugEnabled && _serverStore.IsPassive() == false)
+                    Logger.Debug("Failed to put my node info, will try again later", e);
```

Comment:
I know when we get that it sometimes results in Invalid License. Can we make it a Warn?